File: app/src/main/java/com/rkbapps/neetflix/models/images/PersonImageModel.java
package com.rkbapps.neetflix.models.images;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class PersonImageModel {
    @SerializedName("id")
    @Expose
    private Integer id;
    @SerializedName("profiles")
    @Expose
    private List<Profile> profiles;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public List<Profile> getProfiles() {
        return profiles;
    }

    public void setProfiles(List<Profile> profiles) {
        this.profiles = profiles;
    }
}

File: app/src/main/java/com/rkbapps/neetflix/models/person/images/PersonImageModel.kt
package com.rkbapps.neetflix.models.person.images

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class PersonImageModel(
    @SerializedName("id")
    @Expose
    val id: Int,

    @SerializedName("profiles")
    @Expose
    val profiles: List<Profile>,
)